config.py

```diff
@@ -524,7 +524,9 @@ def get_config(parse_args=True, cfg_path=None, options=None, can_query_registry=
                 agentConfig['statsd_forward_port'] = int(config.get('Main', 'statsd_forward_port'))
 
         if config.has_option("Main", "dogstatsd_remove_aws_resource_tags"):
-            agentConfig["dogstatsd_remove_aws_resource_tags"] = _is_affirmative(config.get("Main", "dogstatsd_remove_aws_resource_tags")) || _is_affirmative(config.get("Main", "dogstatsd_remove_host_tag"))
+            agentConfig["dogstatsd_remove_aws_resource_tags"] = _is_affirmative(config.get("Main", "dogstatsd_remove_aws_resource_tags"))
```

Comment:
why not `or`?

Reply:
We'd have to change the outer guard (`if config.has_option("Main", "dogstatsd_remove_aws_resource_tags")`) to also check for the additional option and then do more conditional checks in here. Seemed easier to just `elif` it.